releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,14 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.1.4/1.2.4 - April 27, 2020
+
+* Add ability to modify client connection to App Configuration and Key Vault. Clients can be provided via ```ConfigurationClientBuilderSetup``` and ```SecretClientBuilderSetup```.
+  * This allows for setting up additional client information such as proxy information.
+* Fixed the bug that caused a partial store to load when both /application/ and /<application_name>/ are used, but one failed to load. This also caused the store to be unable to refresh.
+* Fixed the bug that caused extra requests to be made when a large number of changes existed in the store history.
```

Comment:
What issue did we fix here? Is there a PR for those changes?

Reply:
Fixed a bug that may cause the excessive number of requests being made during configuration change detection to the App Configuration.

(Release notes is to help customers to understand whether an issue applies to them. It does not have to reveal all implementation details. Add an related issue here. It can be useful for anyone to learn more about the issue including implementation details.)